src/amp-tags.js

```diff
@@ -1,6 +1,11 @@
 import get from 'lodash/get';
+import { isStoryPublic } from './utils';
+
+function storyPageAmpTags(story, ampStoryConfig) {
+  if(ampStoryConfig === 'public' && !isStoryPublic(story)) {
```

Comment:
make `ampStoryConfig ` an object?

Reply:
isnt it better to have this check before calling this function itself? here https://github.com/quintype/quintype-node-seo/pull/68/files#diff-a74f76ec0590acc14d75c18234c2d79eR19